src/main/webapp/student-ambassador.html

```diff
@@ -255,6 +243,9 @@ <h5 class="mb-0">
 <script src="assets/js/global-custom-js.js"></script>
 <!-- Script for static navbar and footer -->
 <script src="assets/js/navigation.js"></script>
+<!-- External JS file for the JS functions-->
+<script src="scholarx/2020/functions.js"></script>
```

Comment:
Let's avoid using child/ another parent's assets.

Reply:
my bad we don't need it in the SA page

Reply:
It won't work without the JS function.

Reply:
Ok I resolved this.